test/blueprints/Do.Test.Blueprints.Service.Test/Caching/ScopedMemory.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Extensions.Caching.Memory;
+
+namespace Do.Test.Caching;
+
+public class ScopedMemory : TestServiceSpec
```

Comment:
test case naming should not match class names. to avoid this use heading-like names